docs/learn/protocols/iota2.0/core-concepts/consensus/introduction.md

```diff
@@ -0,0 +1,99 @@
+
+## Required Components for Consensus
+
+The following modules are necessary to achieve agreement using the IOTA 2.0 consensus protocol:
+
+### Approval Weight Module
+
+The approval weight module is responsible for gathering votes from the validation blocks, finding appropriate [validation block patterns](#provide-consensus-flags) in the Tangle, and evaluating the voting results to determine [consensus flags](consensus-flags.md) for blocks and transactions.
+
+:::info Consensus Flags
+
+This article does not specify which algorithms are used to find appropriate validation block patterns; it only focuses on describing the [consensus flags](consensus-flags.md) used by the protocol.
+
+:::
+
+### Tip Selection Algorithm
+
+The tip selection algorithm guides the voting and agreement process, in which the committee and other nodes partake. Consensus is achieved by logically interpreting the locally maintained Tangle, specifically its blocks and the references between them.
+
+All nodes collectively build the Tangle by attaching their blocks to a bounded number of randomly selected [eligible tips](tip-selection-algorithm.md#eligible-tips). Consensus on conflicting transactions is achieved without total ordering in a leaderless way. Committee members cast their votes by regularly issuing validation blocks, encoding their opinions about conflicting payloads by incorporating suitable [block references](preliminaries.md#block-references). The default number of references is increased for validation blocks to improve the time to achieve consensus for blocks and transactions.
+
+### Slot Commitment Chains
+
+Slot commitment chains are one of the core data structures in the protocol. A slot commitment is a hash digest of the accepted blocks and transactions issued during a given slot. It is linked to the commitment of the previous slot, creating a chain that extends back to the genesis block.
+
+Under good network conditions, nodes collectively produce the same slot commitments. Under network partitioning, there could be a divergence in the perception of the suffix of the slot commitment chain. However, the finalized part of the slot commitment chain is safe under any network conditions. When the network converges, all honest nodes eventually switch to the heaviest slot commitment chain with the latest [finalized slot commitment](#finalization), thanks to the **[chain switching rule](chain-switching-rule.md)**.
+
+## The Consensus Protocol’s Two Dimensions
+
+The consensus protocol can be divided in two separate dimensions charged with different responsibilities:
+
+1. [Agreement tasks for blocks and transactions](#agreement-tasks-for-blocks-and-transactions).
+2. [Provide Consensus Flags](#provide-consensus-flags).
+
+### Agreement Tasks for Blocks and Transactions
+
+The first dimension consists of the agreement tasks for blocks and transactions. It has two main tasks:
+
+#### Broadcast blocks reliably
```

Comment:
I would stick with a more standard terminology as it was in the original document "Reliable broadcast of blocks"